sdk/keyvault/azure-security-keyvault-keys/src/key_client.cpp

```diff
@@ -32,7 +30,61 @@ struct RequestWithContinuationToken final
 };
 
 static inline RequestWithContinuationToken BuildRequestFromContinuationToken(
-    Azure::Security::KeyVault::_internal::GetPageResultOptions const& options,
+    Azure::Security::KeyVault::Keys::GetPropertiesOfKeysOptions const& options,
+    std::vector<std::string>&& defaultPath)
+{
+  RequestWithContinuationToken request;
```

Comment:
@vhvb1989 can you help me understand why we needed this change now?

Reply:
We were using the base class as a parameter in a helper function.  I removed the base class.